Filesystem.cpp

```diff
@@ -40,40 +65,40 @@ void FileSystem::touch(vector <string> args){
             return;
         }
     }
-    currentDir->makeFile(args.front());
+    currentDir->makeFile(arg);
     return;
 }
 
-void FileSystem::rm(vector <string> args){
+void FileSystem::rm(string arg){
     if(currentDir->getSubFolders().empty()){  
         cout << "The directory is not exits!"<< endl;
         return;
     }
-    if(!currentDir->searchDir(args.front())->getSubFolders().empty()){
+    if(!currentDir->searchDir(arg)->getSubFolders().empty()){
         cout << "The directory has content in it!"<< endl;
         return;
     }
-    currentDir->remove(currentDir->searchDir(args.front()));
+    currentDir->remove(currentDir->searchDir(arg));
```

Comment:
Tripla keresés: először megkeresi és megnézi, hogy üres-e, majd megkeresi a címet, amit átad a `Directory::remove()`-nak, ami megkeresi a címet és kitörli (mert a `list::remove` is megkeresi az értéket).